assets/sass/utilities/variables/_transition-var.scss

```diff
@@ -0,0 +1,30 @@
+//--------------------------------------------------------------
+// TRANSITIONS
+//--------------------------------------------------------------
+
+////
+/// @group variables
+/// @author WebDevStudios
+////
+
+/// Speed.
+$speed: 0.3s;
+$ease: ease-in-out;
+
+/// Opacity transition.
+$transition-opacity: opacity $speed $ease;
+
+/// Border transition
+$transition-border: border  $speed $ease;
+
+/// All transitions. Use only in emergencies!
+$transition-all: all  $speed $ease;
+
+/// Transform transitions.
+$transition-transform: transform  $speed $ease;
+
+/// Transform Hamburger
+$transition-hamburger: transform $speed, background-color $speed $ease;
```

Comment:
Can we call a transition variable from a transition variable? For instance, instead of doing what we have now do `$transition-hamburger: $transition-transform, $transition-bg-color;`?

Since we're not customizing the speed or easing, I'm wondering if that could work without Gulp getting weird about it.

Reply:
This would work, yeah, as long as the one being called is defined before the other. ie, `$transition-transform` is defined before `$transition-bg-color`